third_party/blink/renderer/core/farbling/brave_session_cache.cc

```diff
@@ -223,11 +223,15 @@ void BraveSessionCache::Init() {
 
 absl::optional<blink::BraveAudioFarblingHelper>
 BraveSessionCache::GetAudioFarblingHelper() {
-  return base::OptionalFromPtr(UpdateAndGetAudioFarblingHelper());
+  if (!farbling_enabled_ || (farbling_level_ == BraveFarblingLevel::OFF))
+    return absl::nullopt;
+  DCHECK(audio_farbling_helper_);
+  return audio_farbling_helper_;
 }
 
 void BraveSessionCache::FarbleAudioChannel(float* dst, size_t count) {
-  if (const auto* helper = UpdateAndGetAudioFarblingHelper()) {
+  if (absl::optional<blink::BraveAudioFarblingHelper> helper =
```

Comment:
previously there was no copy, now there's a copy.
it should be:
```
if (audio_farbling_helper_) {
  audio_farbling_helper_->FarbleAudioChannel(dst, count);
}
```

Reply:
fixed in https://github.com/brave/brave-core/pull/16264/commits/602c9eb34dde0e7560d8455aa3456df91deaadf9